stubs/Pillow/PIL/Image.pyi

```diff
@@ -143,39 +147,37 @@ class Image:
     ) -> Image: ...
     def copy(self) -> Image: ...
     __copy__ = copy
-    def crop(self, box=...) -> Image: ...
-    def draft(self, mode, size) -> None: ...
-    def filter(self, filter): ...
+    def crop(self, box: _Box | None = ...) -> Image: ...
+    def draft(self, mode: str, size: _Size) -> None: ...
+    def filter(self, filter: Filter | Callable[[], Filter]) -> Image: ...
     def getbands(self) -> Tuple[str, ...]: ...
     def getbbox(self) -> tuple[int, int, int, int] | None: ...
-    def getcolors(self, maxcolors: int = ...): ...
-    def getdata(self, band=...): ...
+    def getcolors(self, maxcolors: int = ...) -> list[tuple[int, int]]: ...
+    def getdata(self, band: int | None = ...): ...
     def getextrema(self): ...
     def getexif(self) -> Exif: ...
     def getim(self): ...
-    def getpalette(self): ...
-    def getpixel(self, xy): ...
-    def getprojection(self): ...
-    def histogram(self, mask=..., extrema=...): ...
-    def entropy(self, mask=..., extrema=...): ...
-    def paste(
-        self, im: Image, box: tuple[float, float] | tuple[float, float, float, float] | None = ..., mask: Image | None = ...
-    ) -> None: ...
+    def getpalette(self) -> list[int] | None: ...
+    def getpixel(self, xy: tuple[int, int]): ...
+    def getprojection(self) -> tuple[list[int], list[int]]: ...
+    def histogram(self, mask: Image | None = ..., extrema: tuple[int, int] | tuple[float, float] | None = ...) -> list[int]: ...
+    def entropy(self, mask: Image | None = ..., extrema: tuple[int, int] | tuple[float, float] | None = ...) -> float: ...
+    def paste(self, im: Image, box: tuple[float, float] | _Box | None = ..., mask: Image | None = ...) -> None: ...
     def alpha_composite(self, im: Image, dest: tuple[int, int] = ..., source: tuple[int, int] = ...) -> None: ...
-    def point(self, lut, mode=...): ...
-    def putalpha(self, alpha): ...
-    def putdata(self, data, scale: float = ..., offset: float = ...): ...
-    def putpalette(self, data, rawmode=...): ...
-    def putpixel(self, xy, value): ...
-    def remap_palette(self, dest_map, source_palette=...): ...
+    def point(self, lut, mode: str | None = ...) -> Image: ...
+    def putalpha(self, alpha: Image | int) -> None: ...
+    def putdata(self, data: Sequence[int], scale: float = ..., offset: float = ...) -> None: ...
+    def putpalette(self, data: ImagePalette | bytes | str | Iterable[int] | SupportsBytes, rawmode: str | None = ...) -> None: ...
+    def putpixel(self, xy: tuple[int, int], value: _Color) -> None: ...
+    def remap_palette(self, dest_map: Iterable[int], source_palette: Sequence[int] | None = ...) -> Image: ...
```

Comment:
(No action required) I checked and `Image` (rather than `Self`) is correct: it ultimately calls `_new()` which is hardcoded to return `Image`, not a potential subclass.